aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -333,6 +333,12 @@ module std::features {
         is_enabled(BN254_STRUCTURES)
     }
 
+    /// Whether deploying to objects is enabled.
+    const OBJECT_CODE_DEPLOYMENT: u64 = 48;
+    public fun is_object_code_deployment_enabled(): bool acquires Features {
```

Comment:
Is it possible to enable this in tests by default?

Reply:
Yes, will amend the tests to do so. We can add this into [test features](https://github.com/aptos-labs/aptos-core/blob/36f602a85de0d98a582de4bc3174d45516907c95/aptos-move/vm-genesis/src/lib.rs#L408)